pyne/mesh.py

```diff
@@ -1229,44 +1224,47 @@ def tag_cell_fracs(self, cell_fracs):
         num_vol_elements = len(self)
         # Find the maximum cell number in a voxel
         max_cell_number = -1
-        for ve in range(num_vol_elements):
```

Comment:
Use `i` to replace `ve` for clarity.

Reply:
Why is `i` more clear than `ve`?  Seems the opposite to me.

Reply:
I recommended that something other than `ve` be used here, because `ve` is used [in other places](https://github.com/zxkjack123/pyne/blob/0c9c5b874879d0fa415d4ee35386831afd5634e3/pyne/mesh.py#L725) to represent a volume element rather than index to a volume element, which is what `i` is here. Maybe `ve_idx` could work?

I *think* (though I'm not sure, so maybe @zxkjack123 can comment further) that the normal mesh iterator isn't used here because the index is also needed to access parts of data arrays.

Reply:
@gonuke @pshriwise 
In the pyne/pyne/mesh.py, the function `__iter__(self)`:
```
def __iter__(self):
        """Iterates through the mesh and at each step yield the volume element
        index i, the material mat, and the volume element itself ve.
        """
        mats = self.mats
        if mats is None:
            for i, ve in enumerate(self.iter_ve()):
                yield i, None, ve
        else:
            for i, ve in enumerate(self.iter_ve()):
                yield i, mats[i], ve
```

`i` is more commonly used as a represent of `idx`, while `ve` represents a volume element entity.  
Also, there are lots of 
```
for i, ve in enumerate(ves):
``` 
or 
```
for i, _, ve in mesh:
```
in PyNE codes. 
In this case, I think it's better to use `ve` only when it represents a volume element entity. And use `i` when it represents an index: `idx`.